tests/parsing/mod.rs

```diff
@@ -1,3 +1,4 @@
+use nix::libc::__c_anonymous_ptrace_syscall_info_entry;
```

Comment:
I don't think it's needed

Reply:
Thanks. My IDE must have automatically added this for some reason.